15-7-1_and_15-c-1/15-7-1_and_15-c-1_checks.Rmd

```diff
@@ -0,0 +1,117 @@
+---
+title: "15-7-1 and 15-c-1 checks"
+author: "Katie Uzzell"
+date: "22/12/2022"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+```{r load-packages, include=FALSE}
+library(tidyr)
+library(dplyr)
+library(ggplot2)
+library(kableExtra)
+```
+
+```{r include= FALSE}
+source('15-7-1_15-c-1_example_config.R')
+
+old_data <- read.csv("https://sdgdata.gov.uk/sdg-data/en/data/15-7-1.csv")
+
+date <- Sys.Date()
+new_data <- read.csv(paste0("Example_Output/", date, "_update_15-7-1_15-c-1", ".csv")) 
+
+latest_year <- max(new_data$Year)
+
+```
+
+## 15-7-1 and 15-c-1 update summary
+
+Data for `r min(new_data$Year)` to `r latest_year` have been created and 
+saved as `r paste0(date, "_15-7-1_and_15-c-1", ".csv")` in the 15-1-1 Output folder.  
+    
+### Check configurations
+Please check that the information below is as you expect. In particular, pay 
+attention to the time since data were saved, to check that previously 
+downloaded data have not accidentally been used.
+    
+These settings can all be adjusted in `config.R`
+    
+Files used to create the new data were downloaded and saved in 
+15-7-1_and_15-c-1/`r input_folder`.  
+  
+```{r include= FALSE}
+filename_save_time <- file.info(paste0(getwd(),"/Example_Input/", filename), full.names = TRUE)$ctime[1]
```

Comment:
Again, don't hard-code the input folder, instead:
`filename_save_time <- file.info(paste0(getwd(),"/",input_folder,"/", filename), full.names = TRUE)$ctime[1]`

Reply:
Done